sdk/core/ci.yml

```diff
@@ -12,6 +12,7 @@ trigger:
     - eng/
     exclude:
     - eng/mgmt/
+    - eng/common/
```

Comment:
Did you intend to add this? The description seems to imply you wanted to remove it.

Reply:
I just want to remove the `eng/common` triggering core pipeline instead of other path.

Reply:
This seems backward, though. At least for .NET - and I thought this was intended to be true across language repos - our sdk/core pipeline builds all track 2 projects by design because any changes to core need to be validated across SDKs. So if we *only* run core's pipeline when eng/common we're testing everything already. Running individual projects' pipelines would be equivalent to building core but without actually checking core, which would not be a safe change.

Taking that and my other comment into account, I think the correct strategy would be to *only* run the core pipeline for changes to eng/common and exclude eng/common for every other pipeline.

This is congruent with a conversation some of us had recently about trying to find some way to avoid running individual track 2 projects' pipelines when we run core's pipeline because it does build everything. /cc @JoshLove-msft @christothes @jsquire